### DISCUSSION TITLE
请问如何修改目录里面章节之间的行间距呢？

### DISCUSSION THREAD

**Discussion by bxdd** [nju-lug/NJUThesis#238] [Q&A]
因为目录刚好超出第一页一行，故想修改目录章节的行间距使其更紧凑。想使用 `\setlength{\cftbeforepartskip}{0em}` 但是貌似不管用而且会使得 “目录” 标题二字不居中显示，请问有什么办法呢？

**Comment by atxy-blip** [comment]
呃……同一个问题请不要发两条。GitHub 允许修改已经发出的帖子。

这个问题我建议调调目录的章标题的前后间距，`\tableofcontents` 前后加几句，`0pt` 可以改成你喜欢的数值
```tex
\begingroup
\ctexset{chapter/beforeskip=0pt,chapter/afterskip=0pt}
\tableofcontents
\endgroup
```

**Reply by bxdd** [reply]
不是一个问题呀，这个是求改行间距的qwq

**Reply by atxy-blip** [reply]
改改基线高度硬调吧。反正我更建议按上面的回复，从顶部的章标题空隙找点空间。
```tex
\begingroup
\setlength{\baselineskip}{20pt}
\tableofcontents
\endgroup
```

**Reply by bxdd** [reply]
好的感谢~

**Reply by bxdd** [reply]
修改目录的章标题的前后间距代码并没有生效，但是改baselineskip的却生效了。我其实更想改章节标题艰巨，可以帮忙看看第一个代码哪里有问题吗？

**Reply by atxy-blip** [reply]
第一个代码单纯是改"目录"两个字的前后间距，可以开 `draft` 模式检查效果。
